tests/metaclass_type_test.py

```diff
@@ -0,0 +1,56 @@
+import pytest
+
+from pyupgrade import _fix_py3_plus
+
+
+@pytest.mark.parametrize(
+    's',
+    (
+        pytest.param(
+            'x = type\n'
+            '__metaclass__ = x\n',
+            id='not rewriting "type" rename',
+        ),
+        pytest.param(
+            'def foo():\n'
+            '    __metaclass__ = type\n',
+            id='not rewriting function scope',
+        ),
+        pytest.param(
+            'class Foo:\n'
+            '    __metaclass__ = type\n',
+            id='not rewriting class scope',
+        ),
+        pytest.param(
+            '__metaclass__, __meta_metaclass__ = type, None\n',
+            id='not rewriting multiple assignment',
+        ),
+        pytest.param(
+            '__metaclass__ = (\n'
+            '   type\n'
+            ')\n',
```

Comment:
I think this one can probably be done, just needs to look forward for `NEWLINE` (important-newline token)

Reply:
Updated